headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -240,6 +240,59 @@ struct statuses {
 ASSERT_SIZE(struct statuses, 118);
 #pragma pack(pop)
 
+#pragma pack(push, 1)
+// A bitfield where every bit controls one of the icons that can appear on top of a monster's sprite
+// to represent status effects. If multiple bits are set, the shown icon cycles through them.
+struct status_icon_flags {
+    bool f_sleepless : 1; // Blue eye blinking yellow
+    bool f_burn : 1;      // Red flame
+    bool f_poison : 1;    // White skull
+    bool f_toxic : 1;     // Purple skull
+    bool f_confused : 1;  // Yellow birds
+    bool f_cowering : 1;  // 2 green lines in circle (same as whiffer)
+    bool f_taunt : 1;     // Fist icon
+    bool f_encore : 1;    // Blue exclamation mark (same as low HP)
+    // Blue shield with white sparks. Also for counter, mini counter, mist,
+    // metal burst, aqua ring, and lucky chant
+    bool f_reflect : 1;
+    bool f_safeguard : 1;            // Pink shield. Also for mirror coat
+    bool f_light_screen : 1;         // Golden shield. Also for magic coat
+    bool f_protect : 1;              // Green shield. Also for mirror move and vital throw
+    bool f_endure : 1;               // Blue shield with red sparks
+    bool f_low_hp : 1;               // Blue exclamation mark (same as encore)
+    bool f_curse : 1;                // Red skull
+    bool f_embargo : 1;              // Yellow exclamation mark. Also for gastro acid and snatch
+    bool f_sure_shot : 1;            // Blue sword blinking yellow
+    bool f_whiffer : 1;              // 2 green lines in circle (same as cowering)
+    bool f_set_damage : 1;           // Blue sword blinking red
+    bool f_focus_energy : 1;         // Red sword blinking yellow
+    bool f_blinded : 1;              // Blue eye with an X
+    bool f_cross_eyed : 1;           // Blue question mark
+    bool f_eyedrops : 1;             // Blue eye blinking yellow with a circular wave
+    bool f_muzzled : 1;              // Blinking red cross
+    bool f_grudge : 1;               // Purple shield
+    bool f_exposed : 1;              // Blue eye blinking red with a circular wave
+    bool f_sleep : 1;                // Red Z's
+    bool f_lowered_stat : 1;         // Yellow arrow pointing down
+    bool f_heal_block : 1;           // Blinking green cross
+    bool f_miracle_eye : 1;          // Blinking orange cross
+    bool f_red_exclamation_mark : 1; // Probably unused
```

Comment:
No idea 😆. It doesn't seem to correspond to any normal status condition.

Reply:
I think we can leave it like this then?

Reply:
Yeah it's fine the way it is.